inst/htmljs/animint.js

```diff
@@ -1780,6 +1796,212 @@ var animint = function (to_select, json_file) {
 	return Selectors[s_name].duration;
       })
     ;
+    
+    // selector widgets
+    show_message2 = "Toggle selected variables";
+    var show_hide_selector_widgets = element.append("button")
+      .text(show_message2)
+      .attr("id", "show_hide_selector_widgets")
+      .on("click", function(){
+        if(this.textContent == show_message2){
+          selector_table.style("display", "");
+          show_hide_selector_widgets.text("Hide variable toggler");
+        }else{
+          selector_table.style("display", "none");
+          show_hide_selector_widgets.text(show_message2);
+        }
+      })
+    ;
+    // adding a table for selector widgets
+    var selector_table = element.append("table")
+      .style("display", "none")
+    ;
+    var selector_first_tr = selector_table.append("tr");
+    selector_first_tr
+      .append("th")
+      .text("Toggle selected value");
+      
+    //creating an array to contain the selectized items
+    jquery_thingy_array = [];
```

Comment:
2. I omitted the var here to try and explain what is going on.  If I open the browser, and try to update the `vs` selector, I get this error message.  I'll keep poking around, but if anyone has an idea why this is happening, I would love to hear it.  Also, I apologize for the name.  

![image](https://cloud.githubusercontent.com/assets/6612411/9215321/0981a952-4066-11e5-96f9-6de8e3d413ed.png)